CHANGELOG.md

```diff
@@ -1,5 +1,75 @@
 # go-filecoin changelog
 
+## go-filecoin 0.5.4
+
+We're happy to announce go-filecoin 0.5.4. Highlights include an updated Proof-of-Spacetime implementation and an upgrade-capable network.
+
+### Features
+
+#### 🌳 Network upgrade capability
+
+Two changes have been made to enable software releases without restarting the network. First, a network name is now embedded in the genesis state, permitting multiple networks to follow different upgrade schedules. In addition, the Git SHA compatibility check has been removed from the Hello protocol, enabling nodes with different, but compatible, code to interoperate. Going forward, the user devnet will no longer be restarted with every software release; it will still be restarted as-needed.
+
+#### 🚀 Updated Proof-of-Spacetime (PoSt)
+
+A new proof construction, [Rational PoSt](https://github.com/filecoin-project/specs/blob/master/proof-of-spacetime.md), has been [implemented](https://github.com/filecoin-project/rust-fil-proofs/pull/763) and [integrated](https://github.com/filecoin-project/go-filecoin/pull/3318). This construction is the same shape as our candidate for testnet and resolves outstanding limitations on proving over many sectors.
+
+#### 🎟️ Block and consensus changes
+
+Block headers are now signed by miners, and election tickets form an array in each header.
```

Comment:
The election process is now split into two phases, ticket generation / validation and election winner generation / validation.  Election tickets form an array in each header and mining a null block appends a ticket to this array.  Block headers are now signed by miners.

Reply:
Added, thanks.